exec/lambda/setpositions/dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM public.ecr.aws/lambda/python:3.11-arm64
+FROM public.ecr.aws/lambda/python:3.9-arm64
```

Comment:
are the lambdas running ARM?

Reply:
Yes, the Lambdas are but the SageMaker inference endpoint is AMD (their requirement). The `bin/build_image` script has logic for switching between the target platforms for the image build but that should be reworked/gutted as well.

I'm indifferent as to architecture especially given some of these libraries seem pretty fickle.